prow.sh

```diff
@@ -673,6 +674,29 @@ hostpath_supports_block () {
     echo "${result:-true}"
 }
 
+# The default implementation of this function generates a external
```

Comment:
Can this file be generated from the hostpath driver repo instead of in the general test framework?

Reply:
That would imply also moving everything else related to hostpath out of csi-release-tools. I didn't do that originally because I thought it might be useful to share that default code between repos and/or branches of the hostpath driver repo. I'm not sure whether that reasoning really holds.

But even if it doesn't, that would be a bigger change than I am willing to risk right now. Let's keep it in mind for a future improvement and for now let's keep it here, okay?

Reply:
That's fine. It just seems a bit cumbersome to update how we test hostpath driver this way.